nixos/modules/services/web-apps/peertube.nix

```diff
@@ -370,7 +377,7 @@ in {
     systemd.services.peertube-init-db = lib.mkIf cfg.database.createLocally {
       description = "Initialization database for PeerTube daemon";
       after = [ "network.target" "postgresql.service" ];
-      wantedBy = [ "multi-user.target" ];
+      requires = [ "postgresql.service" ];
```

Comment:
You double checked that this is still started without wantedBy?

Reply:
Checking with nixos test and local virtual machine.I checked work using nixos test and on a local virtual machine.